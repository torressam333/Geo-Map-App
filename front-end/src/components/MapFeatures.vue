<template>
  <section
    class="w-full md:w-auto absolute md:top-[40px] md:left-[60px] z-[2] flex gap-4 px-6 py-8 md:px-0 md:py-0 bg-transparent"
  >
    <div
      class="px-4 bg-white flex items-center shadow-md rounded-md min-h-[50px]"
    >
      <i class="fas fa-location-arrow text-slate-700 text-[1.5rem]"></i>
    </div>
  </section>
</template>

<script>
export default {
  name: 'MapFeatures',
};
</script>
