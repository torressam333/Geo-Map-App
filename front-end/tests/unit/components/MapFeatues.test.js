import { shallowMount } from '@vue/test-utils';
import MapFeatures from '@/components/MapFeatures';

describe('MapFeatures', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(MapFeatures);
  });

  it('renders the modal markup correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
