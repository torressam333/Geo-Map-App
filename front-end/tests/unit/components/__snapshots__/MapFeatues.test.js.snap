// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapFeatures renders the modal markup correctly 1`] = `
<section
  class="w-full md:w-auto absolute md:top-[40px] md:left-[60px] z-[2] flex gap-4 px-6 py-8 md:px-0 md:py-0 bg-transparent"
>
  <div
    class="px-4 bg-white flex items-center shadow-md rounded-md min-h-[50px]"
  >
    <i
      class="fas fa-location-arrow text-slate-700 text-[1.5rem]"
    />
  </div>
</section>
`;
